frontend/src/stylesheets/Home.module.css

```diff
@@ -0,0 +1,266 @@
+
+.textDivider::after {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-top: 3em;
+  margin-left: var(--text-divider-gap);
+}
+
+.centeredContainer {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 3em;
+}
+
+.textDivider2 {
+  display: flex;
+  align-items: center;
+  --text-divider-gap: 1rem;
+}
+
+.textDivider2::before {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-right: var(--text-divider-gap);
+}
+.textDivider2::after {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-left: var(--text-divider-gap);
+}
+
+.imgs {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  padding: 1em;
+}
+
+.imgCaption {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 18rem;
+}
+
+.cert {
+  max-width: 194px;
+}
+
+.caption {
+  text-align: center;
+}
+
+#c1 {
+  margin-top: 2rem;
+}
+
+#c2 {
+  position: relative;
+  bottom: 1.2rem;
+  right: 0.5rem; /* Centers image 2 caption appropriately */
+}
+
+#c3 {
+  margin-top: 1.5rem;
+}
+
+/* Pathways */
+
+h3 {
+  font-size: 24px;
+}
+
+.pathwaysContainer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr); /* 2 columns */
+  grid-template-rows: repeat(2, auto); /* 2 rows */
+  gap: 2rem; /* gap between grid items */
+}
+
+@media screen and (max-width: 1385px) {
+  .pathwaysContainer {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    row-gap: 2rem;
+    align-items: center;
+  }
+}
+
+.pathwayContainer {
+  width: 38rem;
+  height: 400px;
+  border-radius: 10px;
+  border: 1px solid #d8d8d8;
+}
+
+.pathName {
+  margin-top: 1.7rem;
+  margin-left: 1.5rem;
+}
+
+.pathwayText {
+  width: auto;
+  height: 5.5rem;
+  margin: 0.875rem 1.5rem;
+  line-height: 1.5em;
+}
+
+/* FAQ's Title, Contianer, and Components */
+
+.FaqsContainer {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  row-gap: 0.8rem;
+  margin-top: 1.5rem;
+  flex-grow: 1;
+}
+
+.faqBar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 0.7rem;
+  align-items: center;
+}
+
+.faqTitle {
+  width: 1248px;
+  height: 65px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  --text-divider-gap: 1rem;
+  font-size: 28px;
+}
+
+.infoTitle {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 1rem;
+  margin-bottom: 1.5rem;
+}
+
+.infoTitle h2 {
+  align-self: center;
+  margin-top: 0em;
+}
+
+.line {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+}
+
+#answerList {
+  line-height: 1.5;
+}
+
+#pathwaysFAQ {
+  margin-top: 2.5rem;
+}
+/* Exapnd All Component */
+
+.expand {
+  display: flex;
+  width: 145px;
+  height: hug(40px);
+  padding: 8px, 48px, 8px, 16px;
+  border-radius: 4px;
+  border: 1px solid rgba(0, 0, 0, 1);
+  justify-content: center;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.expandAllPlus {
```

Comment:
Styles like this only apply to FAQs so they should be in a file like FAQS.module.css